core/src/execution/proto/expr.proto

```diff
@@ -165,6 +167,18 @@ message CovPopulation {
   DataType datatype = 4;
 }
 
+message VarianceSample {
+  Expr child = 1;
+  bool null_on_divide_by_zero = 2;
+  DataType datatype = 3;
+}
+
+message VariancePopulation {
+  Expr child = 1;
+  bool null_on_divide_by_zero = 2;
+  DataType datatype = 3;
+}
```

Comment:
I don't think we need to have the same struct defined twice here because we have separate field tags (14 and 15) in AggExpr to represent the two different expressions?

Reply:
Thanks for your comment @andygrove 
I am not sure if there is a way to access the field tag. I added an enum `StatisticsType` so I don't need two structs for `VarianceSample` and `VariancePopulation`. Also I think adding a `StatisticsType` is more consistent with the implementation in `variance.rs`.